beam-large-records/test/Test/Record/Beam/Tutorial3.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE ConstraintKinds           #-}
+{-# LANGUAGE CPP                       #-}
```

Comment:
not used?

Reply:
Ah, yes, will remove. Was first going to fix the warning here in a slightly different way.